### ISSUE TITLE
zerotier-cli cannot be used as regular user since 1.4.0 update

### ISSUE THREAD

**Issue by fonix232** [zerotier/ZeroTierOne#996]
**Describe the bug**

After installing zerotier-cli using the official instructions on Ubuntu 19.04, `zerotier-cli` does not function when called as a regular user, and presents the following error:

```
zerotier-cli: missing authentication token and authtoken.secret not found (or readable) in /var/lib/zerotier-one
```

Running it with `sudo` resolves the issue. Adding my main user to the `zerotier-one` group does not alleviate the issue either.

The problem is most likely coming from the default rights of `/var/lib/zerotier-one/authtoken.secret` and `/var/lib/zerotier-one/identity.secret`, which are now limited to user only (chmod 600).

**To Reproduce**
1. Install zerotier-one
2. Try to join a network or request status as a regular user (even if member of sudoers)
3. '....'

**Expected behavior**

`zerotier-cli` should call its daemon running as user `zerotier-one` and return the result

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - OS/Distribution Version: 19.04
 - ZeroTier Version: 1.4.0+
 - Hardware: Any

**Comment by glimberg** [comment]
This is by requirement and design as zerotier interacts with the kernel and requires root privileges in order to do so.  Yes, it runs as the `zerotier-one` user, if present, but it's switched to that after starting as root and dropping the root privileges it doesn't need to operate.

 If you wish, you may `chmod 660`  `/var/lib/zerotier-one/authtoken.secret` and `identity.secret` so that members of the `zerotier-one` group may read them.

**Comment by juju2112** [comment]
@glimberg Couldn't you check if the current user is `root`, and if not, throw an error message saying, "You must be root to run this command"? Then the error message would be more clear.

**Comment by wadewilliams** [comment]
It would be quite nice to add this to the documentation.

**Comment by pcgeek86** [comment]
Please change the error message to indicate that root is required. I'm just learning about ZeroTier for the first time, and didn't realize that root was required. I had to Google a vague error message to figure out what was happening.

As it currently stands, this is a very poor user experience, especially for newcomers. Thankfully, it is an easy fix.

```
zerotier-cli: missing authentication token and authtoken.secret not found (or readable) in /var/lib/zerotier-one
```

**Comment by joseph-henry** [comment]
Fixed in 3870f7af27c9d318e86e33eaf1170b38527d27d9.

Thanks for the suggestion

**Comment by usercopycat** [comment]
i got it working by adding the path to the file. sudo /snap/bin/zerotier join network_number_here
